### PR TITLE
check branch

### PR DESCRIPTION
**Pull request by purple248**
check changes in config.yaml...

### CODE REVIEW

File: data_config.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 #net and train params:
 HIDDEN_SIZE: 16
 BATCH_SIZE: 100
-NUM_EPOCHS: 3
```
Comment (NadavCarmel) [inline data_config.yml]: Why blank line here...

Comment (NadavCarmel) [reply data_config.yml]: Blank line test..

File: data_config.yml [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
\ No newline at end of file
+NUM_EPOCHS: 3
+
+tmp_for_git_test: 1
```
Comment (NadavCarmel) [inline data_config.yml]: Should be 2..

**Comment by purple248** [comment]
closed test pull request

**Comment by purple248** [comment]
reopen to check : Ajusting hyperparams

File: data_config.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 #net and train params:
-HIDDEN_SIZE: 16
-BATCH_SIZE: 100
-NUM_EPOCHS: 3
+HIDDEN_SIZE: 10
+BATCH_SIZE: 50
+NUM_EPOCHS: 4
+LEARNING_RATE: 0.1
```
Comment (NadavCarmel) [inline data_config.yml]: All params make sense here, but keep playing with them and see the effect.
- With small data (like we use) -> better chances of a simple model (narrow hidden layer).
- About learning rate: usually SGD does well with high values (0.1 to 1 more or less), and Adam does well with small values (0.001 to 0.01 more or less). See: https://medium.com/octavian-ai/which-optimizer-and-learning-rate-should-i-use-for-deep-learning-5acb418f9b2

File: simple_nn.py [hunk 4]
```diff
@@ -58,7 +58,7 @@ def forward(self, x):
 
 # Define a Loss function and optimizer
 criterion = nn.MSELoss()
-optimizer = optim.SGD(model.parameters(), lr=0.01)
+optimizer = optim.SGD(model.parameters(), lr=LEARNING_RATE)
 #loader_train = DataLoader(np.concatenate((X_train,y_train.reshape(-1, 1)), axis=1), batch_size=BATCH_SIZE, shuffle=True)
```
Comment (NadavCarmel) [inline simple_nn.py]: great change.  we want to have all configurable parameters in the config file eventually.

File: simple_nn.py [hunk 5]
```diff
@@ -91,13 +91,16 @@ def forward(self, x):
         start_point = end_point
         end_point = BATCH_SIZE * (num_batches + 1)
 
-    y_hat = model.forward(X_train)
+    y_hat = model(X_train)
     print("loss for epoch {}: {}".format(epoch, criterion(torch.tensor(y_train), y_hat.reshape(-1, ))))
 
-    y_hat_test = model.forward(X_test)
+    y_hat_test = model(X_test)
     print("loss for test: {}".format(criterion(torch.tensor(y_test), y_hat_test.reshape(-1, ))))
```
Comment (NadavCarmel) [inline simple_nn.py]: Nice that this works. So now there are no warnings anymore? (yellow code highlight / runtime warnings in the console)
